docs/UsersGuide.asciidoc

```diff
@@ -963,6 +963,10 @@ without needing to install openQA itself. Call `openqa-cli --help` for help.
 The sub-commands provide further help, e.g. `openqa-cli api --help` contains
 a lot of examples.
 
+`openqa-cli` supports auto completion for `bash` and `zsh` with openSUSE/SLE.
+We distribute two packages `openQA-client-bash-completion` and
+`openQA-client-zsh-completion` as part of openQA-client.
```

Comment:
Now you got it the wrong way around. With your contrib files there is auto-completion regardless of openSUSE/SLE but the packages are only available when using the spec file. How about installing within Makefile "install:" instead

Reply:
Can you check again? 

Now `make install` should setup autocompletion according to the $SHELL